### ISSUE TITLE
JAWS doesn't read dynamic text in a Live Region if it is too long

### ISSUE THREAD

**Issue by fwelds322** [FreedomScientific/VFO-standards-support#723]
### Summary
JAWS does not announce new text in a live region when it is displayed. If the text is truncated, it will read it in its entirety.

### Expected result

JAWS read all text in region when shown

### Actual result

JAWS will not read the text if it passes a limit

### Example

https://codepen.io/fwelds322/pen/mdQRLPX

### Additional Information
The current text in the codepen example is too long but if you truncate one char, it will read it all.

#### JAWS version and build number
JAWS 2020.2001.70

 
#### Operating System and version
Windows 10 Enterprise

#### Browser and version:
Chrome 114.0.5735.134
